Lesson9/anna/index.html

```diff
@@ -6,6 +6,395 @@
+                    </g>
+                </g>
+            </g>
+        </g>
+    </g>
+             <button class="action-icon"></button> </svg>
+            
+          <svg width="18px" height="18px" version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 118.28 122.88" style="enable-background:new 0 0 118.28 122.88" xml:space="preserve"><g><path d="M111.85,108.77c-3.47,4.82-8.39,8.52-14.13,10.48c-0.26,0.12-0.55,0.18-0.84,0.18c-0.28,0-0.56-0.06-0.82-0.17v0.06 c0,1.96-1.6,3.56-3.57,3.56l-7.68,0c-1.96,0-3.57-1.6-3.57-3.56l0-55.13c0-1.96,1.6-3.57,3.57-3.57h7.68c1.96,0,3.57,1.6,3.57,3.57 v0.34c0.26-0.12,0.54-0.18,0.82-0.18c0.22,0,0.44,0.04,0.64,0.1l0,0.01c4.36,1.45,8.26,3.92,11.42,7.11V59.15 c0-14.89-4.99-27.63-13.81-36.6l-3.91,5.83c-7.95-8.75-19.4-14.27-32.08-14.27c-12.76,0-24.29,5.59-32.24,14.45l-4.73-5.78 C13.47,31.65,8.54,44.21,8.54,59.15V73.4c3.4-4.08,7.92-7.22,13.07-8.93l0-0.01c0.21-0.07,0.43-0.11,0.64-0.11 c0.28,0,0.57,0.06,0.82,0.17v-0.34c0-1.96,1.61-3.57,3.57-3.57l7.68,0c1.96,0,3.57,1.6,3.57,3.57v55.13c0,1.96-1.61,3.56-3.57,3.56 h-7.68c-1.96,0-3.57-1.6-3.57-3.56v-0.06c-0.25,0.11-0.53,0.17-0.82,0.17c-0.3,0-0.58-0.07-0.83-0.18 c-5.74-1.96-10.66-5.66-14.13-10.48c-1.82-2.52-3.24-5.34-4.17-8.37l-3.12,0V59.15c0-16.27,6.65-31.05,17.37-41.77 C28.09,6.66,42.88,0,59.14,0c16.27,0,31.06,6.66,41.77,17.37c10.72,10.72,17.37,25.5,17.37,41.77v41.25h-2.27 C115.1,103.39,113.68,106.23,111.85,108.77L111.85,108.77L111.85,108.77z"></path></g><button class="action-icon"></button></svg>
+              
+            <svg width="18px" height="18px" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-share"><path d="M4 12v8a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2v-8"></path><polyline points="16 6 12 2 8 6"></polyline></line> <button class="action-icon"></button></svg>
+         
+      </div>   <!svg leri gri yapmaya çalıştım (color="var(--color-secondary)" ama çalışmadı siyah kaldı. class verererek de denedim, html de svg nin içindeki styles kısmından da yaptım ama siyah kaldı. Hover yapıldığında da renk değişimi olmuyor->
+ </div>
+			</section>
+<section class="hero-photo-part">
+  <figure class="hero-photo-parttwo">
+  <img class="hero-content" src="https://miro.medium.com/v2/resize:fit:1100/format:webp/1*aVzJTznRRfP1lM7AXe9yLw.jpeg" alt="Photo by on Unsplash"   </figure>
+    <figcaption class="fig-text">Photo by <a class="fig-text-link" href=""> Kevin Ku</a> on <a class="fig-text-link" href="">Unsplash</a></figcaption>
+	</section>
+  <article class="parag">
+    <p class="scnd">As a web developer, there is a high probability that you have encountered the two enemies of this article:<strong> images </strong> and <strong> deadlines.</strong> Sometimes, for some reasons, your images won’t fit the layout and you don’t want to wrap your head around this for hours.</p>
```

Comment:
Class 'scnd' is used to add space before and after the element. I'd replace it with two classes instead:
```

 .m-t-lg {
    margin-top:var(--space-lg);
  }

  .m-b-xs {
    margin-bottom:var(--space-xs);
  }
```

so in HTML it would be like this:
`<p class="m-t-lg m-b-xs"> `
This way it would be easier to understand what the styling for the element would be without looking into CSS file.